### ISSUE TITLE
Inheritance in metric_namespace and metric_subsystem

### ISSUE THREAD

**Issue by bjb19** [trallnag/prometheus-fastapi-instrumentator#315]
When I set the value of metric_namespace and metric_subsystem at the instrument level it seems like this should propagate down to the metrics (Similar to the default). This does not seem to be the case when working with examples. Happy to open a PR.

**Comment by trallnag** [comment]
Hi, what "examples" do you mean? Can you give an example?

Also feel free to open PR. But note that I have low interest / low priority in this project. So it will be a few weeks until I get around to review, merge, and so on.

**Comment by glebignatieff** [comment]
He means that if you set up your own metrics, then assigning `metric_namespace` and `metric_subsystem` in the `instrument()` method has no effect.

**Comment by bjb19** [comment]
Yeah that is a correct example. Thanks for the notice. This is not a pressing issue for me as there is an easy workaround but I appreciate your time!
